### ISSUE TITLE
Error retrieving Account Data

### ISSUE THREAD

**Issue by electricsunny** [znat/greedy#26]
Added an account, FTX, provided correct API keys and secret.  I see the following error when Greedy tries to load account:

```
TypeError: Cannot read property 'total' of undefined
    at FTXAccount.getUSDBalance (/Users/nssea/.nvm/versions/node/v14.8.0/lib/node_modules/greedy_cli/src/lib/supportedExchanges/FTXAccount.js:38:53)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async Object.header (/Users/nssea/.nvm/versions/node/v14.8.0/lib/node_modules/greedy_cli/src/cli/utils/text.js:34:29)
    at async Object.defaultCommand (/Users/nssea/.nvm/versions/node/v14.8.0/lib/node_modules/greedy_cli/src/cli/commands/default.js:14:5)
  v1.3.2 - Could not connect to account ACCOUNT. Verify your keys.
```

**Comment by znat** [comment]
Hi @electricsunny 
Can you check the contents of your `/Users/nssea/.greedy/profile.yml` and verify your keys are there and are correct?
If they are can you please post the content here, obviously after substituting your keys with dummy values :)

**Comment by znat** [comment]
Issue fixed in 1.3.3
